src/rules.rs

```diff
@@ -624,7 +624,7 @@ pub(crate) fn indentation() -> IndentDsl {
             "#)
         .rule("Indent binops top level")
             .inside(p(NODE_BIN_OP) & p(on_top_level))
-            .not_matching(p(TOKEN_CONCAT) | p(VALUES))
+            .not_matching(p(T![++]) | p(VALUES))
             .set(Indent)
             .test(r#"
                 {
```

Comment:
I think it got deleted in https://github.com/nix-community/nixpkgs-fmt/pull/209/files#diff-a22d0d260e63d7c19b16d3ff4313fd28L633-L647

Reply:
Yeah, but I'd rather merge this PR first, and then rebase that one, as it is more of an experiment at this point

Reply:
Ah yeah you're right, that is make sense.